core/generator.js

```diff
@@ -158,11 +158,12 @@ Blockly.Generator.prototype.allNestedComments = function(block) {
 /**
  * Generate code for the specified block (and attached blocks).
  * @param {Blockly.Block} block The block to generate code for.
+ * @param {boolean} opt_thisOnly True to generate code for only this block.
```

Comment:
The JS Doc (here and below) would be {boolean=} to show it's optional.

Reply:
Also, it should be "this statement" or something to that effect, since a value block would generate code for all its nested blocks, and a loop block would generate code for all its nested statements.